optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/domain/Talk.java

```diff
@@ -46,6 +46,8 @@
     private Set<String> preferredRoomTagSet;
     private Set<String> prohibitedRoomTagSet;
     private Set<String> undesiredRoomTagSet;
+    private Set<String> mutuallyExclusiveTalksTagSet;
+    private Set<String> prerequisiteTalksCodesSet;
```

Comment:
This is an anti-pattern: to use a form of an identifier when we can use a pointer instead. The pointer is always equally fast or faster for queries. Change "Set<String> prerequisiteTalksCodesSet" into "Set<Talk> prerequisiteTalkSet".

Ps: "CodesSet" implies a collection of an array of code (double plural), which it is not.